### PR TITLE
New terms IsISOCurrency & IsUnit

### PR DESCRIPTION
**Pull request by BerSie**
Using these terms it's possible to identify currencies and unit of measures if no amount or price is within the entity. And in addition it eases the determination of currency and unit properties without looking into all ISOCurrency and Unit annotations and check the path there.

### CODE REVIEW

File: vocabularies/Common.xml [hunk 1]
```diff
@@ -155,6 +155,14 @@
         <Annotation Term="Core.RequiresType" String="Edm.String" />
       </Term>
 
+      <Term Name="IsISOCurrency" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Property Parameter">
```
Comment (ralfhandl) [inline vocabularies/Common.xml:158]: Given the lengthy discussions around "internal" currency codes, do we want to have the infix `ISO` here?

Comment (BerSie) [reply vocabularies/Common.xml]: Done

File: vocabularies/Common.xml [hunk 2]
```diff
@@ -155,6 +155,14 @@
         <Annotation Term="Core.RequiresType" String="Edm.String" />
       </Term>
 
+      <Term Name="IsCurrency" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Property Parameter">
+        <Annotation Term="Core.Description" String="Annotated property or parameter is a currency" />
```
Comment (ralfhandl) [inline vocabularies/Common.xml:159]:
```suggestion
        <Annotation Term="Core.Description" String="Annotated property or parameter is a currency code" />

```

**Comment by BerSie** [comment]
New term names are Common.IsCurrency and Common.IsUnit

**Pull request merged**